frontend/app/organiser/event/[id]/page.tsx

```diff
@@ -1,8 +1,58 @@
+"use client";
+
+import EventDrilldownBanner from "@/components/organiser/EventDrilldownBanner";
+import EventDrilldownCommunicationPage from "@/components/organiser/EventDrilldownCommunicationPage";
+import EventDrilldownDetailsPage from "@/components/organiser/EventDrilldownDetailsPage";
+import EventDrilldownManageAttendeesPage from "@/components/organiser/EventDrilldownManageAttendeesPage";
+import EventDrilldownSharePage from "@/components/organiser/EventDrilldownSharePage";
+import EventDrilldownSidePanel from "@/components/organiser/EventDrilldownSidePanel";
+import EventDrilldownStatBanner from "@/components/organiser/EventDrilldownStatBanner";
+import OrganiserNavbar from "@/components/organiser/OrganiserNavbar";
+import { EventData, EventId } from "@/interfaces/EventTypes";
+import { eventServiceLogger, getEventById } from "@/services/src/events/eventsService";
+import { Timestamp } from "firebase/firestore";
+import { useRouter } from "next/router";
+import { useEffect, useState } from "react";
+
+interface EventPageProps {
+  params: {
+    id: string;
+  };
+}
+
 //brians
-export default function EventPage() {
+export default function EventPage({ params }: EventPageProps) {
+  const [currSidebarPage, setCurrSidebarPage] = useState("Details");
+  const [eventData, setEventData] = useState<EventData>();
+
+  const eventId: EventId = params.id;
+  useEffect(() => {
+    getEventById(eventId)
+      .then((event) => setEventData(event))
+      .catch((error) => {
+        console.log("error:", error);
```

Comment:
remove

Reply:
I've addressed this by commenting out this piece of code for the moment.